<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>simulationView</class>
 <widget class="QWidget" name="simulationView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget{
color:#0B0B0B;
font: 11pt &quot;OpenSans-Light&quot;;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="fr_control">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="AcquisitionControl">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="title">
         <string>Acquisition Control</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pb_liveView">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Live View</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_backgroundAcq">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Background Acquisition</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_filterAcq">
           <property name="text">
            <string>Filter Acquisition</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_compute">
           <property name="text">
            <string>Computer, Compute</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="AcquisitionSettings">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="title">
         <string>Acquisition settings</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="la_acqTime">
              <property name="text">
               <string>Acq. time (ms)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_acqTime">
              <property name="text">
               <string>100</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_settings">
           <property name="text">
            <string>Computer, Send settings</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="GraphicsWindow" name="pyqtgraphWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsWindow</class>
   <extends>QGraphicsView</extends>
   <header>graphicswindow.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
